src/_layouts/project-page.html

```diff
@@ -0,0 +1,9 @@
+---
+layout: default
+---
+<header class="project-header">
+  Project
+</header>
+
+<div class="project-collage">
+</div>
```

Comment:
I attempted to configure Jekyll to create product pages, but I failed. I'll need help up product pages.

Reply:
Yeah no worries. The collection mechanism can be a pain in the ass